src/SwiftBindings/tests/CMakeLists.txt

```
@@ -0,0 +1,25 @@
+cmake_minimum_required(VERSION 3.24)
+
+project(SmokeTests)
```

Let's use the Swift language support in CMake and require using a generator that supports it (ie. Ninja). As this code isn't going in the runtime repo, we can be more reasonable about requiring specific features/options.

```suggestion
project(SmokeTests Swift)
```

---

It seems that the `ninja` generator is not installed on build machines: https://github.com/actions/runner-images/blob/macOS-12/20240202.1/images/macos/macos-12-Readme.md

---

> https://github.com/actions/runner-images/blob/macOS-12/20240202.1/images/macos/macos-12-Readme.md

Nit: We use our build images. This link is for GitHub Actions images - I do not think it is relevant to our builds.